earth_enterprise/rpms/opengee-server/post-uninstall.sh

```diff
@@ -0,0 +1,39 @@
+#! /bin/bash
+#
+# Copyright 2017 The Open GEE Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set +x
+
+#-----------------------------------------------------------------
+# Definitions
+GEE="Google Earth Enterprise"
```

Comment:
Open GEE

Reply:
The unused variable has been removed in #612.